tests/components/generic_hygrostat/conftest.py

```diff
@@ -0,0 +1,3 @@
+"""Fixtures for generic_hygrostat tests."""
+
+from tests.components.switch.conftest import mock_switch_entities  # noqa: F401
```

Comment:
We shouldn't import fixtures. It's ok to make a function in a common.py module and define a fixture that uses the function. See `tests/components/conftest.py` for example.